keyboards/hineybush/h60/rules.mk

```diff
@@ -0,0 +1,26 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
```

Comment:
```suggestion
# Bootloader selection
#   Teensy       halfkay
#   Pro Micro    caterina
#   Atmel DFU    atmel-dfu
#   LUFA DFU     lufa-dfu
#   QMK DFU      qmk-dfu
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
```

Reply:
why do these need to be in the code, when I've been told to remove them before? https://github.com/qmk/qmk_firmware/pull/7668#discussion_r359596680